__tests__/__snapshots__/docs.test.js.snap

```diff
@@ -1,140 +1,138 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Testing Docs Renders! 1`] = `
-<div>
```

Comment:
Huh, weird. It could be the renderer... Though I think it's supposed to work with that one. 😕